lib/src/widgets/movie_detail_subtitle.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import '../utils/movie_colors.dart';
+import '../utils/movie_dimensions.dart';
+
+class MovieDetailSubtitle extends StatelessWidget {
+  final String? movieReleaseDate;
```

Comment:
Why this String can be null but in constructor is required?

Reply:
Fixed. Changed from null-able to 'late String' type